docs/standard/library-guidance/dependencies-net.md

```diff
@@ -0,0 +1,169 @@
+---
+ai-usage: ai-assisted
+---
+# Guidance for .NET library dependency versions
+
+When building libraries that target multiple .NET versions, choosing dependency versions has implications for compatibility, servicing, and ecosystem health. This document outlines the main strategies, their tradeoffs, and recommendations.
+
+---
+
+## **Overview**
+
+Library authors face challenges when deciding which version of a .NET dependency to reference. Newer versions have more API and features, but may require a local redistribution increasing servicing responsibilities of the library and size of the application.  The decision impacts:
+
+- **Friction of updates** on older runtimes
```

Comment:
```suggestion
- **Friction of updates** on older runtimes.  Friction might be more changes than are desired by the library directly referenced.  This could be the set of changes introduced between major versions of a dependency, the application size due to more app-local dependencies, the application startup performance due to using app-local dependencies without pre-generated native images, etc.
```